xlinectl/README.md

```diff
@@ -1,3 +1,138 @@
-# Xlinectl
+# xlinectl
 
 This crate provides a command line client for Xline.
+
+## Global Options
+- endpoints <SERVER_NAME ADDR>... -- Set Xline endpoints, which are separated by ','
+    ```bash
+    # connect to servers with specific addresses
+    ./xlinectl --endpoints "server0 10.0.0.1:2379, server1 10.0.0.2:2379, server2 10.0.0.3:2379"
+    ```
+- user <USERNAME[:PASSWD]> -- The name of the user, this provide a shorthand to set password
+    ```bash
+    # connect to servers using user `foo` with password `bar`
+    ./xlinectl --user foo:bar
+    ```
+- password <PASSWD> -- The password of the user, should exist if password not set in `--user`
+    ```bash
+    # connect to servers using user `foo` with password `bar`
+    ./xlinectl --user foo --password bar
+    ```
+- wait_synced_timeout <TIMEOUT> -- The timeout for Curp client waiting synced(in secs) [default: 2]
```

Comment:
Does TIMEOUT need to specify the unit?  `--wait_synced_timeout 2s` and `--wait_synced_timeout 2`, which one is correct?

Reply:
no, it should be consistent with the default type.